### ISSUE TITLE
HTML export leaks reference to origin

### ISSUE THREAD

**Issue by ethrgeist** [hedgedoc/hedgedoc#2767]
### Description

The header section of the exported html contains references back to the origin server.

### Steps to reproduce

1. Open a pad
2. Write something
3. Export as HTML
4. Look into the header section of the file

### Expected behaviour

HTML exports should not need to back reference to the origin domain.

### Logs

```shell
No logs, but the header section.


<head>
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta
    name="viewport"
    content="width=device-width,initial-scale=1,user-scalable=no"
  />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  <meta name="mobile-web-app-capable" content="yes" />
  <title>Domains - HedgeDoc</title>
  <link
    rel="apple-touch-icon"
    sizes="180x180"
    href="https://pad.example.net/icons/apple-touch-icon.png"
  />
  <link
    rel="icon"
    type="image/png"
    sizes="32x32"
    href="{{{url}}}/icons/favicon-32x32.png"
  />
  <link
    rel="icon"
    type="image/png"
    sizes="16x16"
    href="{{{url}}}/icons/favicon-16x16.png"
  />
  <link rel="manifest" href="{{{url}}}/icons/site.webmanifest" />
  <link
    rel="mask-icon"
    href="{{{url}}}/icons/safari-pinned-tab.svg"
    color="#b51f08"
  />
  <link rel="shortcut icon" href="{{{url}}}/icons/favicon.ico" />
  <style>
    [snip bootstrap css]
  </style>
</head>


Also note the not replaced placeholders within the icon references, they should not be there as well.
```

### Config

```shell
environment:
      - CMD_DB_URL=postgres://hedgedoc:hunter2@db:5432/hedgedoc
      - CMD_DOMAIN=pad.example.net
      - CMD_URL_ADDPORT=false
      - CMD_PROTOCOL_USESSL=true
      - CMD_ALLOW_FREEURL=true
      - CMD_SESSION_SECRET=hunter2
      - UPLOADS_MODE=0644
```

### Your Setup

- Host OS: quay.io/hedgedoc/hedgedoc:1.9.4
- NodeJS version: v16.17.1
- HedgeDoc version: 1.9.4

**Comment by davidmehren** [comment]
Can we fix/have we already fixed this in 2.0 @hedgedoc/frontend?

**Comment by DerMolly** [comment]
It's not fixed yet, but we have this tracking issue https://github.com/hedgedoc/react-client/issues/400 for the 2.0 release. If you have anymore request feel free to add them there…
